app/jobs/valkyrie_ingest_job.rb

```diff
@@ -31,11 +33,13 @@ def ingest(file:)
   # @param [Hyrax::FileSet] file_set the file set to add to
   # @param [Hyrax::FileMetadata] file_metadata the metadata object representing
   #   the file to add
+  # @param [::User] user  the user performing the add
   #
   # @return [Hyrax::FileSet] updated file set
-  def add_file_to_file_set(file_set:, file_metadata:)
+  def add_file_to_file_set(file_set:, file_metadata:, user:)
     file_set.file_ids << file_metadata.id
     Hyrax.persister.save(resource: file_set)
+    Hyrax.publisher.publish('object.membership.updated', object: file_set, user: user)
```

Comment:
can remove the todo now

Reply:
unless you want to wait until it's actually rigged up with a listener

Reply:
oh, let's do remove it.